bin/blackbox_edit

```diff
@@ -4,6 +4,7 @@
 # blackbox_edit.sh -- Decrypt a file temporarily for edition, then re-encrypts it again
 #
 set -e
+[ -z $EDITOR ] && EDITOR="vi"
```

Comment:
Please change this to

# If $EDITOR is not set, set it to "vi":
: ${EDITOR:-vi} ;